templates/home/index.html.twig

```diff
@@ -3,7 +3,161 @@
+            <figure>
+                <img src="{{ asset('build/images/suiviTravaux.0aec5935.png') }}" alt="Suivi de travaux" width="42" heigth="42">
+                <figcaption>Suivi de travaux</figcaption>
+            </figure>
+            <figure>
+                <img src="{{ asset('build/images/surveillanceRegulier.4790130a.png') }}" alt="Surveillance régulière" width="42" heigth="42">
+                <figcaption>Surveillance régulière</figcaption>
+            </figure>
+        </div>
+        <div class="textIntendance">
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt<br>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+            </p>
+            <a href="">Plus d'info</a>
+        </div>
+        <a href="">Contact</a>
+    </div>
+</section>
+
+<section class="partenaireParticulier">
+    <h2>Partenaires</h2>
+    <div class="logoPartenaire">
+        <!-- visible part -->
+        <nav>
+            <a href=""><img src="{{ asset('build/images/partenaireAbritel.c4772922.png') }}" alt="Abritel" width="125" heigth="125"></a>
+            <a href=""><img src="{{ asset('build/images/partenaireAirbnb.254afb4d.jpg') }}" alt="Airbnb" width="125" heigth="125"></a>
+            <a href=""><img src="{{ asset('build/images/partenaireBooking.99b19167.png') }}" alt="Booking" width="125" heigth="125"></a>
+        </nav>
+        <input type="checkbox" name="displayMore">
+        <!-- hidden part -->
+        <div class="partenaireInvisible">
+            <a href=""><img src="{{ asset('build/images/partenaireFewo-direkt.a4ce5a8f.png') }}" alt="Fewo-direkt" width="125" heigth="125"></a>
+            <a href=""><img src="{{ asset('build/images/partenaireGites.48dd28f6.jpg') }}" alt="Gites.fr" width="125" heigth="125"></a>
+            <a href=""><img src="{{ asset('build/images/partenaireHomelidays.2079fdda.png') }}" alt="Homelidays" width="125" heigth="125"></a>
+            <a href=""><img src="{{ asset('build/images/partenaireODTAmboiseVaLoire.2fe915a9.jpg') }}" alt="ODT Amboise Val de Loire" width="125" heigth="125"></a>
+            <a href=""><img src="{{ asset('build/images/partenaireODTMontlouisVouvray.dd201d79.png') }}" alt="ODT Mont luis Vouvray" width="125" heigth="125"></a>
+            <a href=""><img src="{{ asset('build/images/partenaireODTToursVaLoire.acc0fd4d.png') }}" alt="ODT Tours Val de Loire" width="125" heigth="125"></a>
+            <a href=""><img src="{{ asset('build/images/partenaireTouraineVaLoire.3edb48fb.jpg') }}" alt="Touraine Val de Loire" width="125" heigth="125"></a>
+            <a href=""><img src="{{ asset('build/images/partenaireValLoireFrance.84836987.png') }}" alt="Val de Loire France" width="125" heigth="125"></a>
+            <a href=""><img src="{{ asset('build/images/partenaireVRBO.cccadb79.png') }}" alt="VRBO" width="125" heigth="125"></a>
+        </div>
```

Comment:
This is also a list of links. It should probably be a nav too or be in the same nav